### PR TITLE
Suggested change to introduce a No Team Type

### PR DESCRIPTION
**Pull request by KaranAtTeamForm**
Suggested change to introduce a No Team Type

- Update diagrams.net TeamTopologies.xml
- Updated Miro files with new shape
- Updated Images/Screenshots
- Updated PPT with new shape

Outside of the proposed shape, we’ve had lots of discussion about the name of the shape and keen for discussion on what we’ve proposed.

### CODE REVIEW

**Comment by conjurer-rich** [comment]
I’m not sure what you are trying to achieve by introducing a “no team type”, can you expand a little?

**Comment by tkidman** [comment]
> I’m not sure what you are trying to achieve by introducing a “no team type”, can you expand a little?

Hi @conjurer-rich - a bit of background for how this change came about. My team has been building a plugin for Miro that makes use of the Team Topology visual language and concepts. A while back we got to demo this to @matthewskelton. In our plugin we drag teams sourced from our TeamForm app onto a Miro board and then we apply a team topology to these teams, but we didn't know how to represent these teams on the board when a team topology had not yet been selected. We came up with the dotted oval shape as found in this PR. Matthew saw this representation of 'no team type' and said it was almost exactly what he had envisaged, and asked us if we could raise a PR against this repo to add it in. 
In short, 'no team type' is meant to represent a team that has yet to have its team topology defined.  Happy to answer any follow up questions.

Cheers, Tom

File: README.md [hunk 1]
```diff
@@ -142,6 +144,7 @@ Where possible the shapes should adhere to the following scheme:
 | Enabling team              |     #DFBDCF     |        0%         |      #D09CB7       |     Solid     |      2px       |
 | Complicated Subsystem team |     #FFC08B     |        0%         |      #E88814       |     Solid     |      2px       |
 | Platform team              |     #B7CDF1     |        0%         |      #6D9EEB       |     Solid     |      2px       |
+| Team                       |     #EBEBEF     |        0%         |      #9B99AF       |    Dotted     |      2px       |
```
Comment (conjurer-rich) [inline README.md:147]: Change "Team" to "Undefined Team Type"

File: README.md [hunk 2]
```diff
@@ -69,6 +70,7 @@ The image below is an example of the shapes you may have seen in the book. You m
 The following team interaction modelling shapes are available:
 
 - All shapes for the four fundamental team types (Stream-aligned, Enabling, Complicated Subsystem, and Platform)
+- A generic team is team with no team type determined
```
Comment (conjurer-rich) [inline README.md:73]: An undefined team type shape can be used to represent a team that currently has no team type determined

File: README.md [hunk 3]
```diff
@@ -38,13 +38,14 @@ Remember these guidelines:
 
 The shapes have been designed with the specific intention of being usable by everyone including those that may have a color vision deficiency. For this reason, the following constraints should apply:
 
-- Platform teams should always have square corners 
+- Platform teams should always have square corners
 - Stream-aligned teams should always be horizontally aligned with rounded corners
 - Enabling teams should always be vertically aligned with rounded corners
 - Complicated subsystem teams should always be an octagon
+- Team should always be horizontally aligned with rounded corners and dotted border
```
Comment (conjurer-rich) [inline README.md:45]: An Undefined Team Type team should always be horizontally aligned with rounded corners and dotted border

**Review by conjurer-rich — COMMENTED** [review]
Also, see general comment on the PR about moving the Undefined Team Type shape under a different column in the diagrams/tools

**Comment by KaranAtTeamForm** [comment]
@conjurer-rich 

Have pushed the changes based on the comments.

**Review by conjurer-rich — CHANGES_REQUESTED** [review]
@KaranAtTeamForm why is the canvas.json a binary file, this should be a plain text json file?
